dotnet/src/Planners/Planners.Core/Handlebars/HandlebarsPlan.cs

```diff
@@ -14,15 +16,23 @@ public sealed class HandlebarsPlan
     private readonly Kernel _kernel;
     private readonly string _template;
 
+    /// <summary>
+    /// Gets the prompt template used to generate the plan.
+    /// </summary>
+    /// <returns>The prompt template.</returns>
+    public string Prompt { get; }
+
     /// <summary>
     /// Initializes a new instance of the <see cref="HandlebarsPlan"/> class.
     /// </summary>
-    /// <param name="kernel">The kernel.</param>
-    /// <param name="template">The Handlebars template.</param>
-    public HandlebarsPlan(Kernel kernel, string template)
+    /// <param name="kernel">Kernel instance.</param>
+    /// <param name="template">A Handlebars template representing the plan.</param>
+    /// <param name="createPlanPromptTemplate">Prompt template used to generate the plan.</param>
+    public HandlebarsPlan(Kernel kernel, string template, string createPlanPromptTemplate)
```

Comment:
Consider adding more details about `template` and `createPlanPromptTemplate` parameters to clearly describe the difference between the two.

Reply:
I added some comments as well as changed the parameter name in the constructor:
![image](https://github.com/microsoft/semantic-kernel/assets/125500434/09abefc2-0786-4ad8-8663-181a4f090ed0)

I didn't want to change the class `_template` property name because ``HandlebarsPlan._template` makes sense